File: 국,영,수 과목 점수를 입력받아 총점과 평균을 계산하고, 등급 판정.cpp
#include <iostream>
#include <iomanip>

using namespace std;

struct st_score {
    int kor, eng, math;
    int total;
    double average;
};

int main() {
    st_score s;
    if (cin >> s.kor >> s.eng >> s.math) {
        s.total = s.kor + s.eng + s.math;
        s.average = s.total / 3.0;

        cout << s.total << " " << fixed << setprecision(1) << s.average << endl;
        cout << "Korean - " << (s.kor >= 70 ? "Pass" : "Fail") << endl;
        cout << "English - " << (s.eng >= 70 ? "Pass" : "Fail") << endl;
        cout << "Math - " << (s.math >= 70 ? "Pass" : "Fail") << endl;
    }
    return 0;
}

File: java/국,영,수 과목 점수를 입력받아 총점과 평균을 계산하고, 등급 판정.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        if (scanner.hasNextInt()) {
            int kor = scanner.nextInt();
            int eng = scanner.nextInt();
            int math = scanner.nextInt();
            int total = kor + eng + math;
            double average = total / 3.0;
            System.out.printf("%d %.1f\n", total, average);
            System.out.println("Korean - " + (kor >= 70 ? "Pass" : "Fail"));
            System.out.println("English - " + (eng >= 70 ? "Pass" : "Fail"));
            System.out.println("Math - " + (math >= 70 ? "Pass" : "Fail"));
        }
    }
}
